data/utils/defectsFromBias.py

```diff
@@ -94,6 +95,6 @@ def writeDefectsFile(bboxList, path, detectorSerial, detectorName):
 
     test2BBoxList = getBBoxList(DefectsPath, detectorName)
     assert len(bboxList) == len(test2BBoxList)
-    for boxA, boxB in itertools.izip(bboxList, test2BBoxList):
+    for boxA, boxB in zip(bboxList, test2BBoxList):
```

Comment:
I wonder why futurize didn't do that automatically. I think it's a tiny list, so it may not matter, but I'm surprised